src/main/java/com/denizenscript/depenizen/bukkit/commands/mythicmobs/MythicSignalCommand.java

```diff
@@ -0,0 +1,91 @@
+package com.denizenscript.depenizen.bukkit.commands.mythicmobs;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.ArgumentHelper;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import com.denizenscript.depenizen.bukkit.objects.mythicmobs.MythicMobsMobTag;
+import io.lumine.xikage.mythicmobs.adapters.bukkit.BukkitAdapter;
+
+import java.util.List;
+
+public class MythicSignalCommand extends AbstractCommand {
+
+    public MythicSignalCommand() {
+        setName("mythicsignal");
+        setSyntax("mythicsignal [<mythicmob>|...] [<signal>] (source:<entity>)");
+        setRequiredArguments(2, 3);
+    }
+
+    // <--[command]
+    // @Name MythicSignal
+    // @Syntax mythicsignal [<mythicmob>|...] [<signal>] (source:<entity>)
+    // @Group Depenizen
+    // @Plugin Depenizen, MythicMobs
+    // @Required 2
+    // @Maximum 3
+    // @Short Sends a signal trigger to the target MythicMobs.
+    //
+    // @Description
+    // This allows you to send a signal trigger to multiple MythicMobs.
+    // If those mobs have any triggers configured for that signal, they will fire.
+    // Optionally, specify an entity that acts as the sender.
+    // NOTE: signals are caps sensitive.
+    //
+    // @Usage
+    // Used to trigger the player's target signal "attack"
```

Comment:
Period at end of sentence, also `the player's target` seems an inaccurate description of the code line